packages/cli/src/commands/deploy.ts

```diff
@@ -37,7 +67,17 @@ export default class Deploy extends Command {
       console.log('Error: no environment name provided. Usage: `boost deploy -e <environment>`.')
       return
     }
+
+    if(flags.skipRestoreDependencies){
+      skipRestoreDependencies = true;
+    }
+
     process.env.BOOSTER_ENV = flags.environment
     await runTasks(flags.environment, compileProjectAndLoadConfig(), deployToCloudProvider)
```

Comment:
Here you add the parameter you've added before, passing `flags.skipRestoreDependencies` directly
```suggestion
    await runTasks(flags.skipRestoreDependencies, flags.environment, compileProjectAndLoadConfig(), deployToCloudProvider)
```